test/integration/integration-test.js

```diff
@@ -140,6 +154,20 @@ describe.skip('integration tests', function() {
       _.partial(verifyResult, 'OfferCreate'),
       _.partial(testTransaction, this, 'order', lastClosedLedgerVersion),
       (txData, callback) => {
+        this.api.getOrders(wallet.getAddress(), {}, composeAsync(orders => {
+          assert(orders && orders.length > 0);
+          const createdOrder = _.first(_.filter(orders, (order) => {
+            return order.properties.sequence === txData.Sequence;
+          }));
+          assert(createdOrder);
+          assert.strictEqual(createdOrder.specification.direction, orderSpecification.direction);
```

Comment:
Would it work to do `assert.strictEqual(createdOrder.specification, orderSpecification);`?